src/main/java/com/qaprosoft/carina/demo/utils/ScreenshotUtil.java

```diff
@@ -17,32 +19,35 @@
 import java.util.GregorianCalendar;
 
 public class ScreenshotUtil implements IDriverPool {
+    String screenName;
+
+    File outputFile = null;
+
     private static final Logger LOGGER = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
+
     public static Calendar calendar = new GregorianCalendar();
+
     public static SimpleDateFormat sdf = new SimpleDateFormat("MM.dd.yyyy-HH.mm");
 
     public static String dateToString(Date date) {
         return sdf.format(date);
     }
 
-    String screenName;
-    File outputFile = new File("reports//screenshots//" + screenName);
-    BufferedImage image = null;
-
-
     public void takeScreenshot() {
         screenName = dateToString(calendar.getTime()) + ".png";
-        getSceenshot();
+        outputFile = new File(R.CONFIG.get("fileScreenshot") + screenName);
```

Comment:
38 and 44 lines are duplicates, why not to add argument to getScreenshot(String fileName) and move lines to this method?